README.rst

```diff
@@ -0,0 +1,323 @@
+Overview
+********
+
+.. _XEN: https://xenproject.org/
+.. _Xen-Troops: https://github.com/xen-troops
+.. _zephyr-xenlib: https://github.com/xen-troops/zephyr-xenlib
+
+A simple `XEN`_ Zephyr Dom0 application **zephyr-dom0-xt** showcasing the use of `Xen-Troops`_
+library `zephyr-xenlib`_ to create and manage `XEN`_ domains from
+Zephyr application by using `zephyr-xenlib`_ API and SHELL interface command.
+
+The **zephyr-dom0-xt** application intended to be started by `XEN`_ as Dom0 domain.
+
+The **zephyr-dom0-xt** dosn't create any `XEN`_ guest domains on start by default.
+
+Requirements
+************
+
+Supported platforms:
+
+- TODO: what is the board for default cfg domd_cfg.c
+- **rpi_5** `Raspberry Pi 5 <https://www.raspberrypi.com/products/raspberry-pi-5/>`_
+
+Prepared Xen guest domains configuration files in ``src/dom_cfg/`` folder.
+
+Prepared Xen guest domains binary image files of kernel and device-tree in ``prebuilt/`` folder.
+
+The Xen quest domains which uses real HW shell be created and started only once even though
+`zephyr-xenlib`_ doesn't prevent it. From another side, the Xen quest domain which doesn't use
+any real HW can be started as many times as `zephyr-xenlib`_ allows using the same Xen guest domains
+configuration.
+
+Devicetree details
+==================
+
+The **zephyr-dom0-xt** on devicetree configuration depends on used board/platform.
+
+Xen guest domains configuration
+*******************************
+
+The **zephyr-dom0-xt** application uses Xen guest domains configuration in `zephyr-xenlib`_
```

Comment:
Wording contains tautology and is hard to read, you can simplify it.
`For Xen domain configuration zephyr-dom0-xt uses ... that is defined in ...`